scrapy_autounit/utils.py

```diff
@@ -215,12 +216,18 @@ def write_test(path, test_name, fixture_name, encoding, url):
 
 class AutoUnit(unittest.TestCase):
     def test__{test_name}__{fixture_name}(self):
+        files = os.listdir(os.path.abspath(Path(__file__).parent))
+        files = [f for f in files if f.endswith('.bin')]
         self.maxDiff = None
-        file_path = (
-            Path(__file__).parent / '{fixture_name}.bin'
-        )
-        test = generate_test(file_path.resolve(), '{encoding}')
-        test(self)
+        for f in files:
+            file_path = (
+                Path(__file__).parent / f
+            )
+            print(
+                "Writing test '{test_name}' for fixture '{fixture_name}'"
+                " in file %s" %f)
```

Comment:
Thanks for looking into this. Actually I meant a message like "Testing fixture x" or something similar. At this point we're not writing any new tests nor fixtures.